### ISSUE TITLE
Morphine Litter creates rtp entries

### ISSUE THREAD

**Issue by Glowbal** [acemod/ACE3#1971]
From a test session last night:
```
21:39:42 z\ace\addons\medical\data\littergeneric_morphine.p3d: No geometry and no visual shape
21:39:42 z\ace\addons\medical\data\littergeneric_morphine.p3d: No geometry and no visual shape
```

**Comment by jokoho48** [comment]
did you use a binarised Build?

**Comment by jokoho48** [comment]
fixed in #1972
